SwiftyBeaverTests/BaseDestinationTests.swift

```diff
@@ -343,9 +343,14 @@ class BaseDestinationTests: XCTestCase {
             path: "/world/ViewController.swift",
             function: "myFunc",
             message: "Hello World"))
+        
+        XCTAssertFalse(destination.shouldLevelBeLogged(.Verbose,
+            path: "/world/ViewController.swift",
+            function: "myFunc",
+            message: "Hello World"))
 
         // not in filter but matching global minLevel
-        XCTAssertTrue(destination.shouldLevelBeLogged(.Info,
+        XCTAssertFalse(destination.shouldLevelBeLogged(.Info,
```

Comment:
Here is how I see the logic of filters:
- Check if a log should happen
  - logLevel < minLevel = log can happen, pass on to filters
  - logLevel >= minLevel = log doesn't happen at all
- If it passes, run filters
  - log goes through filters, if its a contains filter it only shows up if its a match.
  - if its a exclude filter, log show up only if it doesn't match

When filters are set you would expect log messages to only show if they match (contains) or not match (excludes).